tox.ini

```diff
@@ -1,9 +1,9 @@
 [tox]
 # Remember to add to .travis.yml if this is added to.
 envlist =
```

Comment:
might be worth adding 1.10 to the travis config, (as an allowed failure until the final release)